express/index.ts

```diff
@@ -0,0 +1,11 @@
+import express, { Express, Request, Response } from "express";
```

Comment:
use import type for types, and why express twice?

Reply:
fix